src/Makefile.am

```diff
@@ -25,6 +25,10 @@ libt8_installed_headers = \
   src/t8_geometry/t8_geometry_implementations/t8_geometry_linear.hxx \
   src/t8_geometry/t8_geometry_implementations/t8_geometry_zero.hxx \
   src/t8_vec.h src/t8_vtk.h \
+  src/t8_cmesh_netcdf.h \
+  src/t8_forest_netcdf.h \
+  src/t8_element_shape.h \
+  src/t8_netcdf.h \
```

Comment:
@holke: What is the criterion for a header file to be an installed header? Do all of these headers need to be installed?

Reply:
Installed headers headers that should be seen from the outside.
So if it is necessary for user code (e.g. a simulation application) to include this header, then it should be installed.
If the header is only used internally in t8code (and does not contain declarations of function for a user) then it should not be installed but internal.